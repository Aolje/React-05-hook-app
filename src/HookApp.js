import React from 'react'

export const HookApp = () => {
  return (
    <div>
        <h1>Hola universo</h1>
    </div>
  )
}
