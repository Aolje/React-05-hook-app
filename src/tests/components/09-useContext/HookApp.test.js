import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from "../../../HookApp";

describe('Prueba en HookApp', () => {
    test('Debera mostrar el componente correctamente', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
})