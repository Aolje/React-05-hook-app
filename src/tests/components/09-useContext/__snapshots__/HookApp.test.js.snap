// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en HookApp Debera mostrar el componente correctamente 1`] = `
<div>
  <h1>
    Hola universo
  </h1>
</div>
`;
